submissions/damilolaSalami/html-form/contact.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>BuyMe Contact</title>
+</head>
+
+<body>
+    <header>
+        <h1><a href="#">BuyMe</a></h1>
+        <nav>
+            <ul>
+                <li><a href="#">Log In</a></li>
+                <li><a href="#">Create a profile</a></li>
+                <li><a href="productpage.html">Product Page</a></li>
```

Comment:
Well done @queensaldam 

Kindly remove this linking between two pages and add them in your next task i.e Adding navigation links